var/spack/repos/builtin/packages/ginkgo/package.py

```diff
@@ -24,6 +24,7 @@ class Ginkgo(CMakePackage, CudaPackage, ROCmPackage):
     version("master", branch="master")
     version("1.5.0.glu", branch="glu")
     version("1.5.0.glu_experimental", branch="glu_experimental")
+    version("1.5.0", branch="develop")
```

Comment:
```suggestion
    version("1.5.0.glu_experimental", branch="glu_experimental")
    version("1.5.0", branch="develop", preferred=True)
```

I believe this should fix the error which appeared in the CI.

Reply:
Ah - or remove `preferred=True` from 1.4.0? will try that.